ui.content/src/main/content/jcr_root/content/hclecomm/language-masters/en/.content.xml

```diff
@@ -2,6 +2,8 @@
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
     jcr:primaryType="cq:Page">
     <jcr:content
+        cq:cloudserviceconfigs="[/etc/cloudservices/generic-tracker/google-analytics]"
```

Comment:
@ShubhamGarg-AEM : this should not be checked in as part of codebase , but part of authoring guide

Reply:
i have mentioned this in AG also, but since it is a root page this property can be here. Although i can remove this. Should i?